src/box/key_def.h

```diff
@@ -418,9 +423,10 @@ box_key_def_new(uint32_t *fields, uint32_t *types, uint32_t part_count);
  *
  * Default flag values are the following:
  *
- *  | Flag                         | Default value |
- *  | ---------------------------- | ------------- |
- *  | BOX_KEY_PART_DEF_IS_NULLABLE | 0 (unset)     |
+ *  | Flag                          | Default value |
+ *  | ----------------------------- | ------------- |
+ *  | BOX_KEY_PART_DEF_IS_NULLABLE  | 0 (unset)     |
+ *  | BOX_KEY_PART_DEF_EXCLUDE_NULL | 0 (unset)     |
```

Comment:
When the new flag will land to master, please, don't forget to request to document the new changes. This particular API is not documented yet, so you can just leave a note in https://github.com/tarantool/doc/issues/1611. Or you can add it to the docbot request in the commit message.

BTW, it should be `@TarantoolBot document`, not `@documentbot document` if I don't miss something.